test/test-helper.js

```diff
@@ -1,4 +1,94 @@
-const Twilio = require('twilio');
+const Twilio = require("twilio");
+const { fsHelpers } = require("@twilio-labs/serverless-api");
+
+// Heavily lifted from Twilio Run
+// Added instance cache, for test purposes
+// Note: This is very lazy. You must call `global.Runtime.load()` to populate the assets and functions 
+class RuntimeInstance {
```

Comment:
@dkundel @philnash Added this to get access to Runtime.getAssets() when testing files that use it. Would like feedback on usage...will ping in another comment.